packages/modules/auth/src/services/auth-module.ts

```diff
@@ -282,6 +282,74 @@ export default class AuthModuleService
           createdAuthIdentity
         )
       },
+      update: async (
```

Comment:
thought(non-blocking): Would it be more appropriate to call this `updateProviderIdentity`?

The other two methods, `retrieve` and `create`, works on `AuthIdentities`, while as this one works on `ProviderIdentities`.

Reply:
Ah well I guess we pass provider identity data, but it creates an auth identity. Let's keep retrieve and create as-is and only rename the provider identity one so it's not another breaking change.

Reply:
> Let's keep retrieve and create as-is and only rename the provider identity one so it's not another breaking change.

Yeah, this is what I had in mind 👍